install/kubernetes/cilium/templates/cilium-proxy/configmap.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if and .Values.proxy.enabled (not .Values.preflight.enabled) }}
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cilium-proxy-config
+  namespace: {{ .Release.Namespace }}
+data:
+{{- (tpl (.Files.Glob "files/cilium-proxy/configmap/*").AsConfig .) | nindent 2 }}
```

Comment:
multiple files might not work here, so we can just explicitly add filename here.

Reply:
it would work, but might not be what's expected. set the filename of the bootstrap config explicitly.